src/modules/rlm_ldap/rlm_ldap.c

```diff
@@ -1990,13 +2058,24 @@ static int mod_bootstrap(module_inst_ctx_t const *mctx)
 		group_attribute = "LDAP-Group";
 	}
 
-	if (paircmp_register_by_name(group_attribute, attr_user_name, false, rlm_ldap_groupcmp, inst) < 0) {
-		PERROR("Error registering group comparison");
-		goto error;
-	}
-
 	inst->group_da = fr_dict_attr_by_name(NULL, fr_dict_root(dict_freeradius), group_attribute);
 
+	/*
+	 *	If the group attribute was not in the dictionary, create it
+	 */
+	if (!inst->group_da) {
+		fr_dict_attr_flags_t	flags;
+
+		memset(&flags, 0, sizeof(flags));
+		if (fr_dict_attr_add(fr_dict_unconst(dict_freeradius), fr_dict_root(dict_freeradius),
```

Comment:
Hm, this should probably be a tmpl in future definitions so the user can choose where to write out the group info